src/libraries/System.IO.MemoryMappedFiles/tests/MemoryMappedFile.CreateFromFile.Tests.cs

```diff
@@ -868,5 +869,80 @@ public void ReusingNames_Windows(string name)
             }
         }
 
+        private void ValidateDeviceAccess(MemoryMappedFile memMap, long viewCapacity, MemoryMappedFileAccess access)
+        {
+            using (MemoryMappedViewAccessor view = memMap.CreateViewAccessor(0, viewCapacity, access))
+            {
+                if (access != MemoryMappedFileAccess.Write)
+                {
+                    byte b = view.ReadByte(0);
+                    // /dev/zero return zeroes.
```

Comment:
Could we also validate something that doesn't just return 0s, e.g. validate with /dev/urandom that we get back some non-zero values?

Reply:
That is tricky. We cannot write to /dev/(u)random so we would need to split the test. And we can get randomly 0 as well. I can add comment that the test verifies ability to read, not the content it self.